src/explorer/model/MavenProject.ts

```diff
@@ -26,6 +26,7 @@ export class MavenProject implements ITreeItem {
     public pomPath: string;
     public _fullDependencyText: string;
     public conflictNodes: Dependency[];
+    public treeNodes: Dependency[];
```

Comment:
dependencies

Reply:
there is already a get function named dependencies

Reply:
I'm wondering whether getter `dependencies` was acutally called. And there's a bug in line 94, wrongly reading values from plugins.

The name 'treeNodes' is not informative. Let's name it `dependencyNodes`.